src/Client/Sdk/MqttClientFactory.cs

```diff
@@ -54,17 +54,19 @@ public MqttClientFactory (string hostAddress, IMqttBinding binding)
         public async Task<IMqttClient> CreateClientAsync (MqttConfiguration configuration)
```

Comment:
If this is no longer useful, let's just flag this method `[EditorBrowsable(EditorBrowsableState.Never)]` plus `[Obsolete]` so we can properly remove it down the road. Also, let's just `return Task.FromResult(...)` instead of doing this `Yield` which is a bit weird.